src/xdocs/config_filters.xml

```diff
@@ -768,12 +755,6 @@ public static void foo() {
                 single comments.  This requires fewer lines to mark a region, and
                 may be aesthetically preferable in some contexts.
             </p>
-            <p>
-                Usage: This filter only works in conjunction with a <code>FileContentsHolder</code>, since that check makes
```

Comment:
no need to warn user, we will remove that module, and put issue(title and description) a reason. Issue will be referenced in release notes.